### PR TITLE
Change profanity on splash page

### PR DESCRIPTION
**Pull request by the-alex**
Closes GH-274.

Wondering if this alternative wording is in the realm of acceptability for a
government, while remaining adequately demonstrative. What do you think?

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -198,7 +198,7 @@ <h2>Made with love.</h2>
           inspiring me to do this.</small></p>
     </div>
   </section>
-  <template>The boogeyman wrote all changes to the **master server** as he was all out of fucks to give. Thus, the slaves were read-only copies of master.</template>
+  <template>The boogeyman wrote all changes to the **master server** as he was running out of sympathy for these morons. Thus, the slaves were read-only copies of master.</template>
```
Comment (wooorm) [inline src/index.html:201]: I think this is pretty nice! Slightly profane. 🌶 I’ve also been using [`butt`](https://github.com/get-alex/alex/blob/b6283b2545332047348adbd98ceaa037e8a22a61/readme.md#checks) 🍑 as an example. Which do you prefer? I’m up for either!

Comment (the-alex) [reply src/index.html:201]: Oh, much better. Will return to this later and see what I come up with. Don't wanna be too flippant if it's going on the splash page. = P

Comment (the-alex) [reply src/index.html:201]: When I said later, I meant later that day, and didn't followup. Shame on me! Trying again for today.

Comment (wooorm) [reply src/index.html:201]: No problem, take your time!

File: src/index.html [hunk 2]
```diff
@@ -198,7 +198,7 @@ <h2>Made with love.</h2>
           inspiring me to do this.</small></p>
     </div>
   </section>
-  <template>The boogeyman wrote all changes to the **master server** as he was all out of fucks to give. Thus, the slaves were read-only copies of master.</template>
+  <template>The boogeyman wrote all changes to the **master server**, as he was tired of waiting for those buttheads in IT. Thus, the slaves were read-only copies of master.</template>
```
Comment (the-alex) [inline src/index.html:201]: Alright, so here's what's up --
A few of the lines I tried didn't render quite perfectly, so I cycled through util I both had the visually correct highlighting (on my computer /w chrome), and used the word `butt` (here, `buttheads`).

It's a bit awkwardly phrased, but I hope it gets the message across in a _"good enough for government work"_ kind of way.

What do you think?

Comment (wooorm) [reply src/index.html:201]: Nice! I think the text being nonsensical is good!

**Comment by wooorm** [comment]
Looks wonderful, thanks Alex! ✨

**Pull request merged**